stage_tester/locustfile.py

```diff
@@ -0,0 +1,122 @@
+from gevent import monkey; monkey.patch_socket()
+from locust import HttpUser, task, between, run_single_user, events
+import os
+import st
+import requests
+import json
+import configparser
+import random
+import logging
+
+
+conf_file = os.getenv("CONFIG_FILE_PATH", "./dev.conf")
+env_var_prefix = os.getenv("ENV_VAR_PREFIX", "LOAD_TEST_DEV_")
+
+class ApiUser(HttpUser):
```

Comment:
Nitpick, according to PEP8 `APIUser` is preferred over `ApiUser`, although `locust` is ignoring that